src/wallet/rl_wallet/rl_wallet_puzzles.py

```diff
@@ -38,44 +58,113 @@ def rl_puzzle_for_pk(
     opcode_create = ConditionOpcode.CREATE_COIN.hex()
     opcode_myid = ConditionOpcode.ASSERT_MY_COIN_ID.hex()
 
-    TEMPLATE_MY_PARENT_ID = "(sha256 (f (r (r (r (r (r (r 1))))))) (f (r 1)) (f (r (r (r (r (r (r (r 1)))))))))"
```

Comment:
I verified them manually but it isn't a simple 1:1 transform because the new code generator uses the compressed form for accessing arguments which are an integer like 113 instead of (f (r (r (r (r (r (r 1))))))). Also I fixed some bugs along the way